### PR TITLE
adding documentation on how to get codedeploy to work with autospotting

### PR DESCRIPTION
**Pull request by sc-chad**
Wasn't sure where you wanted to link to the documentation so just adding a reference doc.

Closes: https://github.com/cristim/autospotting/issues/14

### CODE REVIEW

**Comment by cristim** [comment]
Thanks for your contribution to the project!

It's fine to have it as a dedicated file, but I think we should reference this wherever we mention CodeDeploy throughout the documentation, there are two places: https://github.com/cristim/autospotting/search?q=CodeDeploy

Just make the 'CodeDeploy' a link in both those occurrences and have it open this page.

**Comment by cristim** [comment]
@sc-chad do you have any time to finish this? I'd like to have this merged.

**Comment by sc-chad** [comment]
Yea I'll try and clean it up today.

**Comment by sc-chad** [comment]
@cristim sorry this took so long.  Finally found time to clean it up.  Thanks again for all the work on it!

File: CODEDEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    -   Save this file to /usr/bin/get-meta on the SPOT AMI to be used 
+
+
+### check-codedeploy 
+
+-   A simple version of a deployment script that is ran on-boot 
+-   This file will need to be deployed to the same SPOT AMI 
+    -   [check-codedeploy](https://gist.github.com/sc-chad/ae0f4acbb5b7283a2dc0b25a3277cf50)
+-   If you are using Amazon Linux saving this file to `/etc/rc3.d/S99deploycode` 
+    -   This will make run it after all networking components are available
```
Comment (xlr-8) [inline CODEDEPLOY.md]: Maybe rephrase it to: "This will make it run after all networking components are available"?

Comment (sc-chad) [reply CODEDEPLOY.md]: This scripts only need to be executable if a non root user is running them or you are trying to run them directly.

Please by all means copy the gists.  I wrote both so only one author.

Comment (sc-chad) [reply CODEDEPLOY.md]: I'm confused.  That's exactly what it says in the latest commit.

Comment (cristim) [reply CODEDEPLOY.md]: @sc-chad I still see lots of markdownlint issues on this file and the URLs still point to the gists in your namespace.

**Comment by cristim** [comment]
@sc-chad please help us get this through.

**Comment by sc-chad** [comment]
I’m out of town I’ll take a look on Monday.

**Comment by cristim** [comment]
@sc-chad do you have any updates?
